docs/docs/Configuration.md

```diff
@@ -8,7 +8,58 @@ index: 5
 Fantomas ships with a series of format options.
 These can be stored in an [.editorconfig](https://editorconfig.org/) file and will be picked up automatically by the commandline tool.
 
-A default .editorconfig file would look like
+<!-- 
+    This index can be auto-generated with VS Code's Markdown All in One extension.
+    The ToC will be updated-on-save, or can be generated on command by using
+    Ctrl-Shift-P: "Create table of contents". 
```

Comment:
Not really a fan of using a format because of the convenience of a certain plugin in a certain editor. My editor (Rider) does something different, so this is a bit unfortunate there is no real standard.

Reply:
@nojaf I agree. But yeah, there's no standard. And of course, other plugins can be used. I'll remove the comment.